src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_BOUND = 100;
```

Comment:
It's good practice to use a non-static context for your methods and variables when they are not meant to be shared across all instances. Consider removing the `static` modifier from `NUMBER_BOUND` unless it's intended to be a class-wide constant.